spec/features/consumer/authentication_spec.rb

```diff
@@ -74,8 +74,8 @@
             expect do
               click_signup_button
               page.should have_content "Welcome! You have signed up successfully"
-            end.to enqueue_job ConfirmSignupJob
-            page.should be_logged_in_as "test@foo.com"
+            end.to enqueue_job Delayed::PerformableMethod
+            Delayed::Job.last.payload_object.method_name.should == :send_on_create_confirmation_instructions_without_delay
```

Comment:
to be honest, considering my lack of recent experience in rspec testing I was taking inspiration from [similar tests](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/models/enterprise_spec.rb#L43). I'll check that out, thanks for the advice!